pkgs/by-name/he/hellwal/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hellwal";
+  version = "1.0.2";
+  src = fetchFromGitHub {
+    owner = "danihek";
+    repo = "hellwal";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-TrqXInoz6OEtS12YmXUILV41IkZW0B4XAAESiU2yMMU=";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+  installPhase = ''
+    install -Dm755 hellwal -t $out/bin
+    mkdir -p $out/share/docs/hellwal
+    cp -r templates themes $out/share/docs/hellwal
+  '';
+  meta = {
+    homepage = "https://github.com/danihek/hellwal";
+    description = "Fast, extensible color palette generator";
+    longDescription = ''
+      Pywal-like color palette generator, but faster and in C.
+    '';
+    license = lib.licenses.mit;
+    platforms = [
+      "x86_64-linux"
+      "x86_64-darwin"
+    ];
```

Comment:
Why x86_64 specifically?

Reply:
Im pretty sure it failed previously before on aarch64, both linux and darwin

Reply:
I'm not sure about `platforms.all` either. Set it to `platforms.unix` for now.

Reply:
Done: [hellwal: init at 1.0.2](https://github.com/NixOS/nixpkgs/pull/364360/commits/64b5ea8e481a379b337ee0959f7e74d6c3e5d6e2)